packages/main/src/compose/newConversation.test.ts

```diff
@@ -0,0 +1,45 @@
+import * as cache from "../cache"
+import { inbox } from "../cache/testFixtures"
+import db from "../db"
+import { MessageInput } from "../generated/graphql"
+import { composeNewConversation } from "./newConversation"
+
+let account: cache.Account
+let accountId: cache.ID
+let boxId: cache.ID
+
+beforeEach(async () => {
+  const { lastInsertRowid } = db
+    .prepare("insert into accounts (email) values (?)")
+    .run("jesse@sitr.us")
+  accountId = lastInsertRowid
+  account = cache.getAccount(accountId)!
+  boxId = cache.persistBoxState(accountId, inbox)
```

Comment:
The function for composing a new conversation does not depend on cache state, so there is no need to call `persistBoxState` or to assign a `boxId` variable.

Actually this test file does not need any database setup at all. You can make up a value for `account` that is compatible with the cache record shape like this:

```ts
const account = { email: "jesse@sitr.us", id: 1 }
```